willow/plugins/pillow.py

```diff
@@ -166,6 +168,26 @@ def set_background_color_rgb(self, color):
 
         return PillowImage(new_image.convert("RGB"))
 
+    @Image.operation
```

Comment:
The second point is we try to keep parity where possible between `PillowImage` and `WandImage`, so the changes would need reflected in `willow.plugins.wand.WandImage`

Reference: https://docs.wand-py.org/en/latest/guide/exif.html - (`image.profiles["EXIF"]`, `images.profiles["ICC"]`)

Reply:
Thanks for pointing this out. I would have missed the Wand implementation completely.

ICC color profiles and EXIF data are not saved to all formats by Wand. Pillow manages to get ICC color profiles embedded in PNG and AVIF files, for example, but Wand does not in its current implementation. Am I missing something here? 

**This works:**
JPEG with ICC color profiles and EXIF data
WebP with ICC color profiles

Reply:
Will have to dig into this later today.
Given Wand is a wrapper around ImageMagick, we are likely limited by that.

Reply:
In the `save_as_jpeg` and `save_as_webp` methods of the `WandImage` class, I use the `.clone()` method, which wasn’t used before here. I think this is necessary, as Wand’s/ImageMagick’s operations are destructive (in contrast to Pillow/PIL) and I want to preserve the original image here.